docs/element-definitions.md

```diff
@@ -2376,9 +2372,9 @@ When the `extension` element is used, it SHOULD include a profile link that prov
 
 For changes that need to make unsafe changes, a custom media type or profile SHOULD be used.
```

Comment:
This is not necessarily review on your branch, but the sentence "For changes that need to make unsafe changes" might need rewording.